ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_path_selection.py

```diff
@@ -178,13 +180,17 @@ def _get_local_interfaces_for_path_group(self, path_group_name: str) -> list | N
 
         return local_interfaces
 
-    def _get_dynamic_peers(self) -> dict | None:
+    def _get_dynamic_peers(self, is_local_pg, ipsec) -> dict | None:
         """
-        TODO support ip_local and ipsec ?
+        TODO support ip_local ?
         """
         if not self.shared_utils.is_wan_client:
             return None
-        return {"enabled": True}
+
+        dynamic_peers = {"enabled": True}
+        if is_local_pg and not ipsec.get("dynamic_peers", True):
+            dynamic_peers["ipsec"] = False
```

Comment:
Should this also be set if there is no ipsec_profile defined on the path-group? (if static_peers == False)

Reply:
I believe so yes. AFAIK, the ipsec policy for dynamic peers comes from key controller, which I believe is added by default.